satpy/readers/cmsaf_claas_v2.py

```diff
@@ -0,0 +1,78 @@
+"""Module containing CMSAF CLAAS v2 FileHandler."""
+
+import datetime
+import pyresample.geometry
+from .netcdf_utils import NetCDF4FileHandler
```

Comment:
IIRC absolute imports are preferred

Reply:
I thought that advice about absolute imports referred to Python 2-style implicit relative imports, which were bad and which were removed in Python 3 (relative imports were introduced in Python 2.5).  PEP 8 appears to permit both absolute and explicit relative imports:

> Absolute imports are recommended (...) explicit relative imports are an acceptable alternative (...) implicit relative imports should never be used and have been removed in Python 3.

An absolute import would be:

```from satpy.readers.netcdf_utils import NetCDF4FileHandler```

Personally I tend to prefer relative imports because it makes it more maintainable if packages are refactored, such as when a collection of modules are moved into a different subpackage, but that consideration appears not to be mentioned by PEP 8.

Reply:
As discussed on slack, both ways are fine.